src/PSPackageProject.psm1

```diff
@@ -93,6 +93,102 @@ function Invoke-PSPackageProjectTest
     )
 
     ## TODO implement calling tests
+    Invoke-BinSkim
+}
+
+function Invoke-BinSkim
+{
+    [CmdletBinding(DefaultParameterSetName='default')]
+    param(
+        [Parameter(ParameterSetName='byPath',Mandatory)]
+        [string]
+        $Path,
+        [Parameter(ParameterSetName='byPath')]
+        [string]
+        $Filter = '*'
+    )
+
+    $sourceName = 'Nuget'
+    Register-PackageSource -ProviderName NuGet -Name $sourceName -Location https://api.nuget.org/v3/index.json -erroraction ignore
+    $packageName = 'microsoft.codeanalysis.binskim'
+    $packageLocation = Join-Path -Path ([System.io.path]::GetTempPath()) -ChildPath 'pspackageproject-packages'
+    Write-Verbose "Finding binskim..." -Verbose
+    $packageInfo = Find-Package -Name $packageName -Source $sourceName
+    if($IsLinux)
+    {
+        $binaryName ='BinSkim'
+        $rid = 'linux-x64'
+    }
+    elseif($IsWindows)
+    {
+        $binaryName ='BinSkim.exe'
+        if([Environment]::Is64BitOperatingSystem)
+        {
+            $rid = 'win-x64'
+        }
+        else
+        {
+            $rid = 'win-x86'
```

Comment:
Do we need this? AzDO does not have a x86 agent.

Reply:
We should allow this locally